src/ert/__main__.py

```diff
@@ -666,6 +666,9 @@ def main() -> None:
         with ErtPluginContext(logger=logging.getLogger()) as runtime_plugins:
             logger.info(f"Running ert with {args} in {os.getcwd()}")
             args.func(args, runtime_plugins)
+    except ErtStoragePermissionError as err:
+        logger.error(str(err))
+        sys.exit(str(err))
```

Comment:
This will happen very early on, so it does not make sense to update the span?
Other blocks include;
```
        span.set_status(Status(StatusCode.ERROR))
        span.record_exception(err)
```

Reply:
The whole point of this was not to propagate this type of PermissionError to azure